components/has/components/enable-profiling/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
```

Comment:
A bit of a nit, but `components/has/components` seems a little redundant. Could we just throw these under base instead, like https://github.com/redhat-appstudio/infra-deployments/tree/main/components/has/base/external-secrets?

Reply:
The patches in the components folder update the deployment spec which has now been moved under the separate environment overlays.  We can't apply patches from outside the directory because of this [security restriction](https://kubectl.docs.kubernetes.io/faq/kustomize/#security-file-foo-is-not-in-or-below-bar).  The alternative was to copy every component overlay into each environment directory which I wanted to avoid.

Reply:
Hmm. Could we call `components` something else then? They're not really components of HAS I guess? Maybe `common`?

Reply:
> I guess Kustomize likes to call it components https://github.com/kubernetes-sigs/kustomize/blob/master/examples/components.md

@johnmcollier  @maysunfaisal  If you want, I can adopt the argoCD `k-components` name.  I'm guessing the convention stands for "kustomize components"?  At least this distinguishes itself from an RHTAP component.

Reply:
sure we can adopt `k-components`, if someone already named it that

Reply:
folder has been renamed